### ISSUE TITLE
Blender texture error

### ISSUE THREAD

**Issue by Fanda555** [TheDuckCow/MCprep#287]
Hello,

I want to start doing minecraft animations but when I clicked on the prep materials button everything turned pink. I hope this can be resolved.

Have a nice rest of the day
![Snímek obrazovky 2022-03-22 190943](https://user-images.githubusercontent.com/102180297/159549273-d269dec9-869f-4b3f-90af-54d5895d174e.png)
.

**Comment by StandingPadAnimations** [comment]
Did you delete the MTL file that was exported alongside the obj file or perhaps deleted a texture folder? also what exporter are you using? The pink means Blender can't find the textures.

 Also I've noticed you're using Blender 2.76. I'd suggest getting Blender 3.1 due to the general improvements (and the new version of Cycles that was added in 3.0) and MCprep is guaranteed to work in newer versions (while it has support for older versions, it's not guaranteed and future features may rely on things added in newer versions of Blender).

**Comment by Fanda555** [comment]
Hello, Is there any way to solve this problem on blender version 2.76?  As an exporter I used mineways.  As for the texture folder, I don't think any has been created.  I didn't even delete the mlt file.

**Comment by Fanda555** [comment]
And when I try to export it using mineways, it shows me this.
![Výstřižek](https://user-images.githubusercontent.com/102180297/160418275-e5726000-c542-417b-8cfa-250707497916.PNG)
.

**Comment by StandingPadAnimations** [comment]
Yeah, 2.7x support is slowly disappearing because of the fast pace Blender updates

**Comment by TheDuckCow** [comment]
Hey @Fanda555 back to the original world, I was going to say try exporting as you had done before, but make sure you export the textures first and don't move the obj file after the export is done (that would break the link to the materials).
